tox.ini

```diff
@@ -1,9 +1,8 @@
 [tox]
-envlist = py{3.8,3.9,3.10,3.11,3.12}
+envlist = py{3.9,3.10,3.11,3.12}
 
 [testenv]
 basepython =
-    py3.8: python3.8
     py3.9: python3.9
     py3.10: python3.10
     py3.11: python3.11
```

Comment:
Shouldn't 3.12 and 3.13 be mentioned here too?

Reply:
It seems I completely forgot about tox.ini or might have accidentally forgot to add my changes to it. I will check at home.
If you want, I can also spend a bit of time to migrate tox.ini into pyproject.toml. I think it is a tiny bit more verbose but would mean one less config file.
I initially hesitated to do, because maybe the project prefers to have the config separated.

Reply:
I'd suggest to only fix tox.ini here to reflect the other changes. We could have the migration to pyproject.toml a new pull request, since it is unrelated to the tz issue you're fixing.